doc/source/cluster/kubernetes/examples.md

```diff
@@ -15,3 +15,4 @@ This section presents example Ray workloads to try out on your Kubernetes cluste
 More examples will be added in the future. Running the distributed XGBoost example below is a
 great way to start experimenting with production Ray workloads on Kubernetes.
 - {ref}`kuberay-ml-example`
+- {ref}`kuberay-gpu-training-example`
```

Comment:
Could you also add an entry to the global table of contents, as in https://github.com/ray-project/ray/blob/4b59dfbe59a143ab8dcc505dad860b4c330b6426/doc/source/_toc.yml#L274 ?

Reply:
Updated.